src/test/resources/allfields/output/input-ruimtereis01/bag/bag-info.txt

```diff
@@ -1,4 +1,5 @@
 Payload-Oxum: 25300962.9
+Is-Version-Of: d5e8f0fb-c374-86eb-918c-b06dd5ae5e71
```

Comment:
I would guess that the checksum (`sha1`) for these resulting bags should change as well in `tagmanifest-sha1.txt`? If the test doesn't fail due to that, probably I made a mistake a while back 😄 
Please let me know if this is the case!

Reply:
Yes you are right, tests didn't fail :)

Reply:
Surprise! That's correct; the test shouldn't fail. See the comment in the following code snippet... Please change the `tagmanifest-sha1.txt` files manually.

https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/1f81280cfb6bacfb43ad34aff7ab8086f656f374/src/test/scala/nl.knaw.dans.easy.multideposit/SplitMultiDepositAppSpec.scala#L179-L184